_quarto.yml

```diff
@@ -83,6 +84,10 @@ website:
           - learning-development/r.qmd
           - learning-development/git.qmd
           - learning-development/python.qmd
+          - ADA/ada.qmd
```

Comment:
The title of this page is now sticking out a bit, could we use the acronym and go for `ADA and databricks` or just `Analytical Data Access (ADA)` and then leave databricks as there's already now 3 other databricks L+D pages?

![image](https://github.com/user-attachments/assets/b950a38b-4177-4c19-9aff-ba3477fa278f)